stream.go

```diff
@@ -127,23 +176,38 @@ func (s *Stream) packetize(raw []byte, ppi PayloadProtocolIdentifier) []*chunkPa
 	i := uint16(0)
 	remaining := uint16(len(raw))
 
+	// From draft-ietf-rtcweb-data-protocol-09, section 6:
+	//   All Data Channel Establishment Protocol messages MUST be sent using
+	//   ordered delivery and reliable transmission.
+	unordered := ppi != PayloadTypeWebRTCDCEP && s.unordered
+
 	var chunks []*chunkPayloadData
 	for remaining != 0 {
 		l := min(s.association.myMaxMTU, remaining)
-		chunks = append(chunks, &chunkPayloadData{
+		chunk := &chunkPayloadData{
 			streamIdentifier:     s.streamIdentifier,
 			userData:             raw[i : i+l],
```

Comment:
Let's change `l` to something else -- I had to look several times before I realized it wasn't a 1

Reply:
Ok, will do!

Reply:
Fixed at e928d.  `l` => `fragmentSize`